tlv/tlv_types_generated.go

```diff
@@ -1012,1012 +1012,5032 @@ func (t *tlvType100) tlv() {}
 
 type TlvType100 = *tlvType100
 
+type tlvType101 struct{}
```

Comment:
Q: didn't follow the tlv package for a while...but I think we should only have two new types added here? why there are way more types here🤔

Reply:
see the `numberOfTypes uint32 = 300` line :) 

bumped from 100 -> 300 so as to include 160 which is the start of the un-signed range

Reply:
FWIW, you can always just make targeted types. They just need to be in this package. You could make a like `custom_types.go` file, then define like: 
```
type TlvType50001 = *tlvType50001

etc
```